loops_lab.js

```diff
@@ -0,0 +1,304 @@
+
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs
+// all the numbers, descending, between "num" and 1.
+//
+
+// // let num = 15
+// // while (num > 0){
+// //   console.log(num)
+// //   num -=1
+// // }
+// // //
+// //
+// // for (let num = 15; num > 0; num -=1)
```

Comment:
Missing { } after for loops. It works in this case since we only have one if statement to run, but you will encounter problems in the future if loop is doing more than one action.